docs/en/query_language/agg_functions/reference.md

```diff
@@ -996,6 +996,44 @@ FROM ontime
 └─────────────────────┘
 ```
 
+## topKWeighted {#topkweighted}
+
+Returns an array of the most frequent values in the specified column. The resulting array is sorted by weight of value. If the optional parameter `N` is passed, сreates a slice of array with n elements.
```

Comment:
Wrong.

Slice is always created. If parameter N is unspecified, default value 10 is used.